components/centraldashboard/public/components/main-page.css

```diff
@@ -78,60 +82,38 @@ app-drawer .footer>.build {
     color: rgba(255, 255, 255, 0.64)
 }
 
-#NamespaceSelector {
-    position: absolute;
-    right: 1em;
-    top: 50%;
-    transform: translateY(-50%)
-}
-
 app-toolbar {
-    display: flex;
-    justify-content: center;
+    @apply --layout-horizontal;
+    @apply --layout-center;
     background: white;
     transition: background .25s;
 }
 
-app-toolbar[blue] {
-    color: white;
-    background: var(--primary-background-color);
-    --primary-text-color: white;
-    --secondary-text-color: #ffffff69;
-    --primary-color: white;
-    --accent-color: white;
-    --logo-color: white;
-}
-
 app-toolbar>header {
     margin-top: auto;
 }
 
 #Narrow-Slider {
-    position: absolute;
-    left: 0;
-    bottom: 0;
-    top: 0;
-    width: 256px;
+    @apply --layout-horizontal;
+    @apply --layout-center;
+}
+
+#Narrow-Slider[hidden] {
```

Comment:
`[hidden]` should be a global style by itself, not linked to any one element

Reply:
Understood. However, the specificity of applying the flex layout to `#NarrowSlider` overrides the `hidden` style.

![Screenshot from 2019-05-06 09-27-04](https://user-images.githubusercontent.com/6835846/57228267-5ff92900-6fe1-11e9-890f-942fa4c49c90.png)

Also, using opacity: 0 still claims the normal amount of space so it doesn't result in the other elements sliding over to occupy the space the logo would have been in.

Reply:
I see, yes the opacity: 0 thing was only for fade in and out, which only should happen on `[hides][hidden]` not on a default `[hidden]`